### PR TITLE
using latest api

### PR DESCRIPTION
**Pull request by ekerwin**

### CODE REVIEW

File: src/main/java/com/blackducksoftware/integration/hub/service/model/ComponentVersionPolicyViolationCount.java [hunk 1]
```diff
@@ -46,8 +46,8 @@ public ComponentVersionPolicyViolationCount(final NameValuePairView nameValuePai
             name = PolicySeverityType.valueOf(nameValuePair.name);
         }
```
Comment (ekerwin) [inline src/main/java/com/blackducksoftware/integration/hub/service/model/ComponentVersionPolicyViolationCount.java]: Not in love with this, but hub-common-api has (correctly) changed value from String to Object, and Gson converts it to a Double. Any other ideas?

Comment (jamesrichard91) [reply src/main/java/com/blackducksoftware/integration/hub/service/model/ComponentVersionPolicyViolationCount.java]: Are you against something like this?
` if (nameValuePair.value != null) {
            value = ((Double) nameValuePair.value).intValue();
         }`

Comment (jamesrichard91) [reply src/main/java/com/blackducksoftware/integration/hub/service/model/ComponentVersionPolicyViolationCount.java]: If your worried about the casting, them I like your approach for now

File: src/main/java/com/blackducksoftware/integration/hub/api/view/MetaHandler.java [hunk 2]
```diff
@@ -40,7 +40,7 @@ public MetaHandler(final IntLogger logger) {
     }
 
     public boolean hasLink(final HubView view, final String linkKey) throws HubIntegrationException {
-        final ResourceMetadata meta = view.meta;
+        final ResourceMetadata meta = view._meta;
```
Comment (psantos1113) [inline src/main/java/com/blackducksoftware/integration/hub/api/view/MetaHandler.java]: To remain consistent with the Hub you have removed the serialized name annotation, is that a correct assumption here?

File: src/main/java/com/blackducksoftware/integration/hub/service/model/ComponentVersionPolicyViolationCount.java [hunk 3]
```diff
@@ -46,8 +46,8 @@ public ComponentVersionPolicyViolationCount(final NameValuePairView nameValuePai
...
 
-        if (StringUtils.isNumeric(nameValuePair.value)) {
-            value = Integer.valueOf(nameValuePair.value);
+        if (nameValuePair.value != null && NumberUtils.isCreatable(nameValuePair.value.toString())) {
```
Comment (psantos1113) [inline src/main/java/com/blackducksoftware/integration/hub/service/model/ComponentVersionPolicyViolationCount.java]: can you extract nameValuePair into a variable before the if statement then use use that might make it a little easier to read.

Object countJsonObject = nameValuePair.value;
if(countJsonObject != null && NumberUtils.isCreatable(countJsonObject.toString()) {
    value = NumberUtils.createNumber(countJsonObject.toString()).intValue();
}

**Pull request merged**